source/manual/fall-back-to-backup-cdn.html.md

```diff
@@ -0,0 +1,36 @@
+---
+owner_slack: "#re-govuk"
+title: Fall back to backup CDN
+section: Deployment
+layout: manual_layout
+parent: "/manual.html"
+---
+
+There two backup Content Delivery Network (CDN) that can be used if Fastly is down.
+AWS CloudFront or Google Cloud CDN.
+
+**Important** At the time of writing, GOV.UK will be served
+by static mirrors and not the origin of GOV.UK if you fallback to one of the backup CDN.
+
+## Procedure
+
+To switch to AWS Cloudfront:
+You will have to make 2 DNS changes to GOV.UK:
+
+1. you will have to ask JISC (very few people are authorised to make such changes) to change
+   the cname of `www.gov.uk` to the `www` AWS CloudFront distribution domain.
+   You can log into the AWS web console to find the `www` AWS CloudFront distribution domain.
```

Comment:
Maybe we could say how to do this from the CLI, to save people having to work out exactly where to look for the domain?

I think this would do it:

```
gds aws govuk-production-readonly \
  aws cloudfront list-distributions --query "DistributionList.Items[?Comment=='WWW'].DomainName"
```